app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,11 @@
+<% provide(:title, "About") %>
+<h1>About</h1>
```

Comment:
EM tìm hiểu `I18n in rails`

Sau đó áp dụng I18n vào tất cả các file `html` nhé